java/core/src/main/java/com/whylogs/core/resolvers/StandardResolver.java

```diff
@@ -0,0 +1,52 @@
+package com.whylogs.core.resolvers;
+
+import com.whylogs.core.DataTypes;
+import com.whylogs.core.metrics.Metric;
+import com.whylogs.core.metrics.StandardMetric;
+import com.whylogs.core.schemas.ColumnSchema;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class StandardResolver extends Resolver {
+  public StandardResolver() {
+    super();
+  }
+
+  // TODO: the rest of the metrics need implmeented
+  // TODO: does this Metric loose it's typing?
+  @Override
+  public HashMap<String, ? extends Metric> resolve(ColumnSchema schema) {
```

Comment:
Note: this doesn't have the column_name passed as it is never used in any python resolver

Reply:
For Dataschema override where user specifies type overrides for a column, not used in standard resolvers, so fine to leave off at first if you want, but its part of the python interface for that reason.

Reply:
Thank you :)